Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerQuoteBubbleView.swift

```diff
@@ -8,44 +8,66 @@ import UIKit
 public typealias ChatMessageComposerQuoteBubbleView = _ChatMessageComposerQuoteBubbleView<NoExtraData>
 
 open class _ChatMessageComposerQuoteBubbleView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
-    // MARK: - Properties
-    
-    public var avatarViewSize: CGSize = .init(width: 24, height: 24)
-    public var attachmentPreviewSize: CGSize = .init(width: 34, height: 34)
-    
-    public var message: _ChatMessage<ExtraData>? {
+    /// Default avatar view size. To override this, subclass `setUpLayout` and add width and height constraints.
+    private var avatarViewSize: CGSize = .init(width: 24, height: 24)
+
+    /// Default attachment preview size. To override this, subclass `setUpLayout` and add width and height constraints.
+    private var attachmentPreviewSize: CGSize = .init(width: 34, height: 34)
```

Comment:
I've made this private, but probably is not the best idea, especially if you just want to change the size of the avatar and the attachments. If the customer changes the size in the `setUpLayout` the hiding will not work properly, since it will use the old width and height.